tests/testHtmIndex.py

```diff
@@ -218,7 +226,7 @@ def testIngest(self):
         default_config.mag_column_list = ['a', 'b']
         default_config.mag_err_column_map = {'a': 'a_err', 'b': 'b_err'}
         default_config.dataset_config.ref_dataset_name = 'myrefcat'
-        default_config.dataset_config.indexer.active.depth = 10
+        default_config.dataset_config.indexer.active.depth = self.depth - 1
```

Comment:
I'm puzzled by this change and by the old code. The old code used self.depth = 8 (any idea why?) which was not self.depth - 1. 

Please add a code comment explaining why this depth is chosen.